docs/docs/transactions.md

```diff
@@ -119,7 +119,57 @@ return sequelize.transaction().then(function (t) {
 });
 ```
 
-# Using transactions with other sequelize methods
+# Options
+The `transaction` method can be called with an options object as the first argument, that
+allows the configuration of the transaction.
+
+```js
+return sequelize.transaction({ /* options */ });
+```
+
+The following options (with it's default values) are available:
+
+```js
+{
+  autocommit: true,
+  isolationLevel: 'REPEATABLE_READ',
+  deferrable: undefined
+}
+```
+
+The `isolationLevel` can either be set globally when initializing the Sequelize instance or
+locally for every transaction:
+
+```js
+// globally
+new Sequelize('db', 'user', 'pw', {
+  isolationLevel: Sequelize.Transaction.ISOLATION_LEVELS.SERIALIZABLE
+});
+
+// locally
+sequelize.transaction({
+  isolationLevel: Sequelize.Transaction.ISOLATION_LEVELS.SERIALIZABLE
```

Comment:
It should be _s_equelize (lower case), so the user is able to use the sequelize instance. This applies to all of these calls.